pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -53,6 +53,18 @@ import (
 const (
 	// DefaultBootstrapTokenTTL default ttl for bootstrap token
 	DefaultBootstrapTokenTTL = 10 * time.Minute
+
+	// MachineRegionLabelName as annotation name for a machine region
+	MachineRegionLabelName = "failure-domain.beta.kubernetes.io/region"
```

Comment:
all this should belong to `machine.openshift.io/*`

Reply:
Why do you want to create a new label name when there is already one existing for exactly the same purpose?

Reply:
kubernetes.io namespace ones are reserved and for nodes, these might or might not mean the same and follow a different lifecycle

Reply:
As you say, life-cycle of the labels for nodes is different from life-cycle for machines (which is not yet defined). Are you afraid of confusion?

Reply:
> all this should belong to `machine.openshift.io/*`

Is this change for our own consumption? Asking because I see this in the kube tree:

```
* This change applies zone labels to vSphere Volumes automatically. The zone labels are visible on the PV: ([#72687](https://github.com/kubernetes/kubernetes/pull/72687), [@subramanian-neelakantan](https://github.com/subramanian-neelakantan))
    * $ kubectl get pv --show-labels
    * NAME           CAPACITY   ACCESSMODES   STATUS    CLAIM            REASON    AGE       LABELS
    * pv-abc                5Gi        RWO                   Bound     default/claim1                    46s       failure-domain.beta.kubernetes.io/region=VC
```

Should we support both - where both is a new label and but also propagate to the existing label?